sql/create-docs.sh

```diff
@@ -42,7 +42,7 @@ pushd "$FWDIR" > /dev/null
 # Now create the markdown file
 rm -fr docs
 mkdir docs
-echo "Generating markdown files for SQL documentation."
+echo "Generating SQL documentation: markdown files + config table"
```

Comment:
no big deal but I would write like "Generating markdown files for SQL documentation, and HTML files for SQL configuration list" just to be more explicit.

Reply:
And just to be more explicit, we can fix https://github.com/apache/spark/blob/master/sql/README.md too (like https://github.com/apache/spark/pull/18702/files#diff-af3ff743d6b643f25cdd5670173d3daeR12)

Reply:
and updating https://github.com/apache/spark/pull/18702/files#diff-a4b1e8e0e72fd59bd246285a34b21a45R20 here too.

Reply:
and comment above :-)

Reply:
Will do, and I will probably also rename `gen-sql-markdown.py` so the name reflects its new purpose.